tools/src/test/java/org/apache/kafka/tools/reassign/ReassignPartitionsCommandTest.java

```diff
@@ -432,6 +433,21 @@ topicPartition, new PartitionReassignmentState(asList(0, 1, 2), asList(0, 1, 2),
         }
     }
 
+    @ClusterTest
+    public void testDisallowReplicationFactorChange() {
```

Comment:
Could you also add a test (or build on this one) which showcases that **increasing** the replication factor also fails?

Reply:
Sure!